### ISSUE TITLE
Earth Defense Force 2 portable d3d11 hardware transform lightning issue(?)

### ISSUE THREAD

**Issue by LunaMoo** [hrydgard/ppsspp#10172]
Here's with hardware transform ON:
![uljs00374_00000](https://user-images.githubusercontent.com/5485237/33119010-efbf9cb0-cf6d-11e7-90dc-569d1e0d9b98.jpg)
And here's with it turned off(or any other backend with HW transform ON as it looks about same).
![uljs00374_00001](https://user-images.githubusercontent.com/5485237/33119026-fc894acc-cf6d-11e7-94e5-9486e4b9ca93.jpg)

I think everything renders it's just soo dark it looks black? Doesn't affect any other backend.

Including a dump which doesn't show sky, but that doesn't happen in the actual game:
[ULJS00374_0001.zip](https://github.com/hrydgard/ppsspp/files/1495285/ULJS00374_0001.zip)

**Comment by hrydgard** [comment]
Yup, looks like a lighting issue - you can see in RenderDoc in the mesh output that all vertices get a color 0,0,0 after transform.

Oh yeah BTW, this is that crazy game that changes culling mode after nearly every draw, which results in a trillion little draw calls, reducing performance...  Only way I can see around that is a special mode for this where we reverse the triangle strip indices when culling is backwards, instead of actually switching the culling mode. Good to have the dump for that, whenever anyone gets around to it...

**Comment by unknownbrackets** [comment]
Any change here?

Has this ever worked right in Direct3D 11?

-[Unknown]

**Comment by LunaMoo** [comment]
No change and I believe it never worked in d3d11 with hw transform.
